lte/gateway/python/integ_tests/s1aptests/test_mobile_reachability_tmr_with_mme_restart.py

```diff
@@ -20,16 +20,20 @@
 from s1ap_utils import MagmadUtil
 
 
-class TestMobileReachabilityTimerWithMmeRestart(unittest.TestCase):
+class TestMobileReachabilityTmrWithMmeRestart(unittest.TestCase):
```

Comment:
WHat's the motivation to rename the test and file?

Reply:
File and function names are modified to simply suppress the python flake8 formatting warnings for longer names greater than 45 characters. Class name is modified just to keep in sync with the the test file/function name.

s1aptests/test_mobile_reachability_timer_with_mme_restart.py:0:1: WPS118 Found too long name: test_mobile_reachability_timer_with_mme_restart > 45
s1aptests/test_mobile_reachability_timer_with_mme_restart.py:32:5: WPS118 Found too long name: test_mobile_reachability_timer_with_mme_restart > 45